### PR TITLE
♻️ update the copyright year.

### PR DESCRIPTION
**Pull request by shaunchander**
I noticed the copyright year says "2021", updated it to say "2022".

### CODE REVIEW

File: src/common/OurUsers.mjs [hunk 1]
```diff
@@ -121,6 +121,12 @@ var companies = [
     url: "https://camelohq.com",
     path: "/static/lp/camelo.svg",
     [Symbol.for("name")]: "Logo"
+  },
+  {
+    name: "DevIT Jobs",
```
Comment (ryyppy) [inline src/common/OurUsers.mjs:126]: was this an intentional change? if so, could you factor that into its own PR?

Comment (shaunchander) [reply src/common/OurUsers.mjs:126]: No this change was unintentional, not sure how that happened though 😅.

**Comment by robinweser** [comment]
Why don't we just use `new Date().getFullYear()` instead? Then we don't have to change that every year :P

**Comment by shaunchander** [comment]
> Why don't we just use `new Date().getFullYear()` instead? Then we don't have to change that every year :P

I'll add this in.

**Comment by Grubba27** [comment]
I saw this issue and i've done the function... https://github.com/rescript-association/rescript-lang.org/pull/519/files, if it is better I can just place it here and shaunchander can use in this pr :)

**Comment by ryyppy** [comment]
Fixed in #521, due to it not having unwanted file diffs in there. Will check what's up with the `package-lock.json` changes.

Thanks for contributing!
